plugins.json

```diff
@@ -395,6 +395,18 @@
         "remote_install": true,
         "compatible_version": ">=1.0.37"
     },
+    "Greenwing": {
+        "title": "Greenwing",
+        "version": "1.2.9",
+        "author": "Confexion",
+        "license": "MIT",
+        "description": "Greenwing is a kanboard beautify theme with a modern design.",
+        "homepage": "https://github.com/Confexion/Greenwing",
+        "readme": "https://github.com/Confexion/Greenwing/blob/master/README.md",
+        "download": "https://github.com/Confexion/Greenwing/archive/master.zip",
```

Comment:
Please, follow the instructions regarding the archive: https://github.com/kanboard/website#how-to-add-a-new-plugin-to-the-list